stdlib/jdbc/src/test/resources/test-src/actions/select_test.bal

```diff
@@ -272,7 +272,7 @@ function testCloseConnectionPool(string jdbcURL) returns @tainted (int) {
     return count;
 }
 
-function getTableCountValColumn(table<ResultCount> | error result) returns int {
+function getTableCountValColumn(table<record {}> | error result) returns int {
```

Comment:
Why do we need these changes? (and other similar)

Reply:
Earlier, with table type-checking the constraint was not considered. Thus, it was possible to assign a value of type `table<record {}>` (or some table broader than `table<Foo>`) to a variable of `table<Foo>` (as mentioned in https://github.com/ballerina-platform/ballerina-lang/issues/17808). 

This PR fixes the type-checking bug, which requires this change to accept a result of a db client remote method.